### ISSUE TITLE
Get error when I access Store()

### ISSUE THREAD

**Issue by ali3assi** [oxigraph/oxigraph#188]
I installed `pyoxigraph `using `pip install pyoxigraph`

When I reference store like:
```
from pyoxigraph import *
from pyoxigraph import Store
store = Store()
```
I get the following exception:
```
---------------------------------------------------------------------------
ImportError                               Traceback (most recent call last)
~\AppData\Local\Temp/ipykernel_4604/3945786218.py in <module>
      1 from pyoxigraph import *
----> 2 from pyoxigraph import Store
      3 store = Store()

ImportError: cannot import name 'Store' from 'pyoxigraph' (C:\Users\Admin\miniconda3\envs\oxi\lib\site-packages\pyoxigraph\__init__.py)
```

I dont know if I need to install or activate something else to be able to access Store()

**Comment by Tpt** [comment]
Hi! Thank you for trying Oxigraph and sorry for that.

I made a mistake while pushing the first beta for the upcoming 0.3 version: the release script updated the website documentation even if the default package on pypy is still from the 0.2 branch. [I have reverted this change](https://oxigraph.org/pyoxigraph/stable/index.html).

In v0.2, there is no `Store` class but a [`MemoryStore`](https://oxigraph.org/pyoxigraph/v0.2.5/store/memory.html) if you want the data to stay in memory and a [`SledStore`](https://oxigraph.org/pyoxigraph/v0.2.5/store/sled.html) if you want data to be stored on disk. v0.3 merges the two classes into a single `Store` class, hence the documentation discrepency.

Sorry again.

**Comment by ali3assi** [comment]
@Tpt Thanks for your reply. So, finally, the following option doesnt exist in python's wrapper:
`RocksDbStore: a file system implementation based on the RocksDB key-value store. It requires the "rocksdb" feature to be activated. The clang compiler needs to be installed to compile RocksDB.`

Can you please confirm? If yes, do you have any estimation for when it will be added?

**Comment by Tpt** [comment]
@ali3assi yes, `RocksDbStore` is not available in pyoxigraph 0.2. However it is going be available under the name `Store` in the upcoming 0.3 version. I already pushed [a first beta on Pypy](https://pypi.org/project/pyoxigraph/0.3.0_beta.2/) and [the documentation is here](https://oxigraph.org/pyoxigraph/v0.3.0-beta.2/). Feel free to try it and give me feedbacks. I am dropping the other "stores" ("rust in-memory" and Sled) in v0.3 to ease the development and maintenance.

**Comment by ali3assi** [comment]
@Tpt Thanks for you quick reply. Yes it is working now. I wanna tested and give you later my feedbacks
